addons/point_of_sale/static/src/js/Screens/ClientListScreen/ClientDetailsEdit.js

```diff
@@ -47,7 +48,7 @@ odoo.define('point_of_sale.ClientDetailsEdit', function(require) {
             if ((!this.props.partner.name && !processedChanges.name) ||
                 processedChanges.name === '' ){
                 return this.showPopup('ErrorPopup', {
-                  title: _('A Customer Name Is Required'),
+                  title: _t('A Customer Name Is Required'),
```

Comment:
In OWL component it is better to use `this.env._t` instead, as we're trying to remove the require from `web.core`.